src/index.tsx

```diff
@@ -82,6 +82,11 @@ const Router: FunctionComponent = () => {
 													<Route exact path={ROUTES.AIRDROP} component={lazy(() => import('./pages/airdrop'))} />
 													<Route exact path={ROUTES.STAKE} component={lazy(() => import('./pages/home'))} />
 													<Route exact path={ROUTES.VOTE} component={lazy(() => import('./pages/proposals'))} />
+													<Route
+														exact
+														path={`${ROUTES.VOTE}/:proposalId`}
+														component={lazy(() => import('./pages/proposals/proposal-details/details'))}
```

Comment:
Can we move proposal details to its own folder? `pages/proposal-details`

Reply:
updated